libs/eo/landing-page/shell/src/lib/eo-landing-page.integration.spec.ts

```diff
@@ -38,7 +45,21 @@ describe('EnergyOrigin landing page', () => {
       ],
     });
     await navigate('/');
+  });
 
+  it('displays the EnergyOrigin logo', async () => {
     expect(await findEnergyOriginLogo()).toBeInTheDocument();
   });
+
+  it('displays the Energinet logo', async () => {
+    expect(await findEnerginetLogo()).toBeInTheDocument();
+  });
+
+  it('displays a telephone link', async () => {
+    expect(await findPhoneLink()).toHaveAttribute('href');
+  });
+
+  it('displays an e-mail link', async () => {
+    expect(await findEmailLink()).toHaveAttribute('href');
+  });
```

Comment:
suggestion: Verify call phone number and write email functionality in links.
```suggestion
  it('displays a telephone link', async () => {
    expect((await findPhoneLink()).getAttribute('href')).toMatch(/^tel:/);
  });

  it('displays an e-mail link', async () => {
    expect((await findEmailLink()).getAttribute('href')).toMatch(/^mailto:/);
  });
```